fr/sidebars.json

```diff
@@ -927,56 +927,65 @@
             }
         ],
         "Intégrations": [
+            "integrations/introduction-integrations",
             {
                 "ids": [
-                    "integrations/external/accedian-pvx-skylight",
-                    "integrations/external/monitoring-dem-mip"
+                    "integrations/data-analytics/sc-elastic-events",
+                    "integrations/data-analytics/sc-elastic-metrics",
+                    "integrations/plugin-packs/procedures/applications-graylog-restapi",
+                    "integrations/data-analytics/sc-splunk-metrics",
+                    "integrations/data-analytics/sc-splunk-events",
+                    "integrations/data-analytics/sc-warp10"
                 ],
-                "label": "Externes",
+                "label": "Data Analytics",
                 "type": "subcategory"
             },
             {
                 "ids": [
-                    "integrations/notifications/telegram"
+                    "integrations/plugin-packs/procedures/applications-monitoring-alyvix-restapi",
+                    "integrations/plugin-packs/procedures/applications-monitoring-iplabel-datametrie-restapi",
+                    "integrations/plugin-packs/procedures/applications-monitoring-mip-restapi",
+                    "integrations/digital-experience-monitoring/pack-skylight-pvx"
                 ],
-                "label": "Notifications",
+                "label": "Digital Experience Monitoring",
                 "type": "subcategory"
             },
             {
                 "ids": [
-                    "integrations/open-tickets/bmc-footprints",
-                    "integrations/open-tickets/bmc-remedy",
-                    "integrations/open-tickets/easyvista",
-                    "integrations/open-tickets/glpi",
-                    "integrations/open-tickets/glpi-restapi",
-                    "integrations/open-tickets/itop",
-                    "integrations/open-tickets/iws-isilog",
-                    "integrations/open-tickets/jira",
-                    "integrations/open-tickets/mail",
-                    "integrations/open-tickets/otrs-restapi",
-                    "integrations/open-tickets/request-tracker-restapi",
-                    "integrations/open-tickets/serena",
-                    "integrations/open-tickets/servicenow"
+                    "integrations/event-management/sc-hp-bsm",
+                    "integrations/event-management/sc-hp-omi",
+                    "integrations/event-management/sc-opsgenie",
+                    "integrations/event-management/sc-pagerduty",
+                    "integrations/event-management/sc-servicenow",
```

Comment:
should be converted to sc-service-now-events as soon as the right file is in place (see previous review comment about replacing doc in french doc)